app/src/main/java/org/oppia/app/home/topiclist/TopicListAdapter.kt

```diff
@@ -31,14 +28,7 @@ class TopicListAdapter(
 ) :
   RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
-  private val orientation = Resources.getSystem().configuration.orientation
-  private val metrics = DisplayMetrics()
-  private var screenWidth = 0
-
-  init {
-    activity.windowManager.defaultDisplay.getMetrics(metrics)
-    screenWidth = metrics.widthPixels
-  }
+  var spanCount = 0
```

Comment:
This should be `private var spanCount = 0` and create a setter function to update this value.

Reply:
From what I know, Kotlin internally creates a setter and a getter and uses them. If you create a Java class and tried to access the `spanCount`, you will only be able to access it using its getter `getSpanCount()` which had been generated automatically.  
PTAL [here](https://www.programiz.com/kotlin-programming/getters-setters)

Reply:
@MohamedMedhat1998 Yeah that is correct but we should avoid making variables public to other files as much as possible and thats why the separate setter function will make things precise.

Reply:
Ok, I'll fix that

Reply:
Done